cli-plugins/socket/socket.go

```diff
@@ -107,9 +109,14 @@ func (pl *PluginServer) closeAllConns() {
 	pl.mu.Lock()
 	defer pl.mu.Unlock()
 
+	if pl.closed {
+		return
+	}
+
 	// Prevent new connections from being accepted.
 	pl.closed = true
 
+	logrus.Debug("Closing all connections")
```

Comment:
I'd prefer not to log this on every plugin invocation with `--debug`

Reply:
Oh, I had this in another place with slightly different wording locally, just forgot to push.